src/transform/WidenImage.js

```diff
@@ -25,6 +25,9 @@ const widenAncestors = el => {
     if (parentElement.style.float) {
       parentElement.style.float = 'none'
     }
+    if (parentElement.style.display) {
```

Comment:
I love that you've taken the trouble to identify a use case in the wild. Would you mind adding a reference here? I've been using a style like: https://en.m.wikipedia.org/wiki/Picasso's_Blue_Period?oldid=806483472.

Reply:
Updated the comment.